ONSdigital/florence
# Release/1.21.0

### What

Release 1.21.0 branch includes:

- Bump `dp-table-builder-ui` version `1.0.20` => `1.1.0` - new version includes table row/col header validation
- Removed chart palette options to ensure accessible chart colours

### How to review

- Ensure changes are below are as described
- Ensure all is ready to be released

### Who can review

Anyone
